src/lib/ctrl.ts

```diff
@@ -65,8 +65,12 @@ export class Ctrl {
                 new TextDecoder().decode(data.slice(4, PACKAGE_SIZE))
             )
         }
-        if (data[2] == MessageType.CONFIG_GIVE) {
-            return new CtrlConfigGive(data[4], data[5])
+        if (data[2] == MessageType.CONFIG_SHARE) {
+            return new CtrlConfigShare(
+                data[4],
```

Comment:
Comment what are the different array indices
https://github.com/inputlabs/alpakka_firmware/docs/ctrl_protocol.md